lib/members/member.ex

```diff
@@ -0,0 +1,40 @@
+defmodule GlassFactoryApi.Members.Member do
+  @moduledoc """
+  Defines the Member of organization.
+  """
+
+  alias GlassFactoryApi.Members.Member
+
+  @type t() :: %Member{
+          name: String.t(),
+          email: String.t(),
+          id: integer(),
+          archived: boolean(),
+          capacity: non_neg_integer(),
+          freelancer: boolean(),
+          joined_at: String.t()
+        }
+
+  defstruct [:name, :email, :id, :archived, :capacity, :freelancer, :joined_at]
+
+  @doc """
+  Parse a map into a Member struct
+
+  ## Examples
+
+      iex> GlassFactoryApi.Member.to_struct(%name: "John Doe", id: 100, email: "john.doe@example.org")
```

Comment:
- missing `{` in the function argument: `to_struct(%{name: ...})`
- 🎨I don't think you _have_ to write the full namespaced path for the function, doin' something like `iex> to_struct(%{name: ...})` should be enough 🤔